examples/apps/table-detail-tea/README.md

```diff
@@ -0,0 +1,11 @@
+# Table detail view
+
+This example shows how to create a table of items and a detail view.
+
+It uses the Elm architecture or TEA approach as described on [the ratatui website](https://ratatui.rs/concepts/application-patterns/the-elm-architecture/).
```

Comment:
This will make markdownlint angry due to line length

Fix:
```suggestion
It uses [the Elm architecture(TEA)](https://ratatui.rs/concepts/application-patterns/the-elm-architecture/).
```